gno.land/pkg/gnoweb/components/ui/icons.html

```diff
@@ -217,5 +217,25 @@
         d="M10.9999 12L3.92886 19.0711L2.51465 17.6569L8.1715 12L2.51465 6.34317L3.92886 4.92896L10.9999 12ZM10.9999 19H20.9999V21H10.9999V19Z"
       />
     </symbol>
-  </svg>
+    <symbol id="ico-moon" viewBox="0 0 24 24">
```

Comment:
Where does those icons come from? What licence? 
Anyway, we don't need icons without JS.